app/_locales/en/messages.json

```
@@ -1035,6 +1035,21 @@
     "message": "File import not working? Click here!",
     "description": "Helps user import their account from a JSON file"
   },
+  "flaskExperimentalText1": {
+    "message": "Using Flask can greatly increase your risk of fund loss:"
+  },
+  "flaskExperimentalText2": {
+    "message": "if you use it to install non-trustworthy Snaps"
```

Despite it being lowercase in the design I think our guidelines are to use sentence case.

```suggestion
    "message": "If you use it to install non-trustworthy Snaps"
```

---

Yes. Let's use sentence case and let's include a period at the end of the sentence. 

I'm not sure if Snaps should be capitalized, tho. We can twitch that in the future.

---

To @rekmarks point it's a continuation of the sentence so lowecase works here. Please ignore.